packages/odyssey-storybook/src/components/Banner/Banner.mdx

```diff
@@ -1,5 +1,5 @@
 import { Canvas, Story, ArgsTable } from "@storybook/addon-docs";
-import { theme } from "@okta/odyssey-react/dist/components/Banner/Banner.theme";
+import { theme } from "../../../../odyssey-react/src/components/Banner/Banner.theme";
```

Comment:
We may want to add a dev docs note to let people know they will need to use relative paths instead of package imports.

Reply:
@edburyenegren-okta good thought yea some type of lint rule would be good

Reply:
What happened to the `@okta` part? Is it because this is part of a monorepo?

In a monorepo, I wouldn't recommend referencing other packages with `../../` since you can't always rely on them being there like that. They should also be in your `package.json`. Also, yarn workspaces provides the auto-linking capabilities.